scipy/stats/unuran/unuran_wrapper.pyi

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+from typing import TYPE_CHECKING, Union, Any, Callable, Tuple, Optional, List
+
+if TYPE_CHECKING:
+    import numpy.typing as npt
+
+
+SeedType = Union[np.random.RandomState, np.random.Generator, int,
+                 npt.ArrayLike]
+
+
+__all__: List[str]
+
+
+class Method:
+    ...
+
+
+class TransformedDensityRejection(Method):
+    def __init__(self,
+                 pdf: Callable, dpdf: Callable,
+                 params: Tuple[Any] = ...,
+                 domain: Optional[Tuple[float, float]] = ...,
```

Comment:
```suggestion
                 domain: None | Tuple[float, float]] = ...,
```
Style suggestion: with [pep 604](https://www.python.org/dev/peps/pep-0604/) being supported by pretty much all type checkers these days, you can use the pipe operator for representing unions. Generally this just looks a bit cleaner.

Reply:
Done, thanks!